src/CorsMiddleware.php

```diff
@@ -167,19 +211,19 @@ private function buildSettings(ServerRequestInterface $request, ResponseInterfac
     {
         $settings = new CorsSettings();
 
-        $origin = array_fill_keys((array) $this->options["origin"], true);
+        $origin = array_fill_keys($this->options["origin"], true);
         $settings->setRequestAllowedOrigins($origin);
 
         if (is_callable($this->options["methods"])) {
             $methods = (array) $this->options["methods"]($request, $response);
         } else {
-            $methods = $this->options["methods"];
+            $methods = (array) $this->options["methods"];
         }
         $methods = array_fill_keys($methods, true);
         $settings->setRequestAllowedMethods($methods);
 
         $headers = array_fill_keys($this->options["headers.allow"], true);
-        $headers = array_change_key_case($headers, CASE_LOWER);
```

Comment:
Did PHPStan complain about this? I prefer having the CASE_LOWER there so it is easy to know what is happening without going to PHP docs to see what is the default behaviour.

Reply:
It did not (but rector will).
I added a comment line above to describe what's happening in here. Adding default-parameters just to document the methods behavior seems a bit odd to me. In IDEs, the methods signature is just a single click away.